analysis/measures_definition.py

```diff
@@ -162,40 +169,45 @@
         denominator=has_appointment & has_registration & (age >= 18),
         group_by={
             "age_greater_equal_65": age_greater_equal_65,
-            "age_65_69": age_65_69,
-            "age_70_64": age_70_64,
-            "age_75_79": age_75_79,
-            "age_80_84": age_80_84,
-            "age_greater_equal_85": age_greater_equal_85,
         },
         intervals=weeks(12).starting_on(start_date),
     )
 
     measures.define_measure(
-        name=f"has_f2f_{time_description}_weekly_age",
-        numerator=has_f2f_consultation,
-        denominator=has_appointment & has_registration & (age > 18),
-        group_by={"age_greater_equal_65": age_greater_equal_65,
-            "age_65_69": age_65_69,
-            "age_70_64": age_70_64,
-            "age_75_79": age_75_79,
-            "age_80_84": age_80_84,
-            "age_greater_equal_85": age_greater_equal_85,
+        name=f"has_virtual_{time_description}_weekly_age_band",
+        numerator=has_virtual_consultation,
+        denominator=has_appointment & has_registration & (age >= 18),
```

Comment:
I'm still not sure if the `has_appointment` variable should be in the denominator. did you discuss this with your team and came to the conclusion that this is what you want?

Reply:
The team would like appointments data as denominator. But we need some more analysis to make this a conclusive thought i.e summary of appointments by statuses.